src/hooks/useWearables.ts

```diff
@@ -110,5 +72,54 @@ export const useSetWearableState = () => {
       .then((res) => res.data);
   };
 
-  return setWearableState;
+  const setLastSync = async ({ ehrId, lastSync, failureCode }: SetLastSync) =>
+    httpClient.patch(
+      `/v1/wearables/${ehrId}`,
+      {
+        lastSync,
+        status: failureCode
+          ? WearableIntegrationStatus.Failure
+          : WearableIntegrationStatus.Syncing,
+        failureCode,
+      },
+      { headers: accountHeaders },
+    );
+
+  const setSyncTypes = async (settings: SyncTypeSettings) =>
+    httpClient.post('/v1/wearables/sync-types', settings, {
+      headers: accountHeaders,
+    });
+
+  const useWearableIntegrationQuery = (ehrId: string) =>
+    useQuery(
+      'get-wearable',
+      () =>
+        httpClient.get(`/v1/wearables/${ehrId}`, { headers: accountHeaders }),
+      {
+        enabled: !!accountHeaders,
+      },
+    );
+
+  const useWearableIntegrationsQuery = (appId: string) =>
```

Comment:
I feel like we should just use the bundleId as the `appId` and remove this argument. This should make things consistent and avoid some confusion on what value should be provided.

Reply:
Ahh, yes, I will do that in a follow up PR. 👍

Reply:
Done